frontend/src/model/__tests__/ConvertDataFormat.test.js

```diff
@@ -1,30 +1,39 @@
+import { expect, test } from "@jest/globals";
 import {
   convertToDataObject,
   convertToServiceObject
 } from "../ConvertDataFormat";
 import DeviceInfo from "../DeviceInfo";
-import {
-  sampleInputChannels,
-  sampleOutputChannels
-} from "../../api/SampleData";
+import InChannelInfo from "../InputChannelInfo";
+import OutChannelInfo from "../OutputChannelInfo";
 
+const sampleInputChannels = [
+  new InChannelInfo(1, "test input ch 1", 500, null),
+  new InChannelInfo(2, "test input ch 2", 456, null),
+  new InChannelInfo(3, "test input ch 3", 800, null)
+];
+const sampleOutputChannels = [
+  new OutChannelInfo(1, "test output ch 1", 500, null),
+  new OutChannelInfo(2, "test output ch 2", 456, null),
+  new OutChannelInfo(3, "test output ch 3", 800, null)
+];
 const sampleLocalSender = new DeviceInfo(
   "test sender serial",
   null,
   "test sender ip",
   "test sender display",
   "offline",
-  sampleOutputChannels,
-  ["Additional Device Info goes here."]
+  undefined,
+  undefined
```

Comment:
Why are these not being assigned values anymore?
Also, if you're going to not set something, use null, not undefined. something should never be defined as undefined

Reply:
The concern you pose is part of a greater issue that needs to be fixed in coordination with the backend in another PR. These fields do not match the data representation being sent between service and frontend, and I "labelled" them undefined to highlight this fact.